extensions/extensions-alerting/extensions-alerting-teams/pom.xml

```diff
@@ -13,9 +13,29 @@
     <name>aiSSEMBLE::Extensions::Alerting::Teams</name>
     <description>The Alerting Teams module uses Microsoft Teams to send alerts</description>
 
-    <properties>
-        <version.smallrye.config.override>2.8.0</version.smallrye.config.override>
-    </properties>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-universe-bom</artifactId>
```

Comment:
I: An extremely common issue with the upgrade was that we were importing the `bom-component` POM and not the Quarkus BOM.  Across the board it worked better to add the Quarkus BOM and entirely remove `bom-component` as it had conflicts with the Quarkus BOM.